### ISSUE TITLE
Multi language for model

### ISSUE THREAD

**Issue by tuong-olli** [keithito/tacotron#49]
How to build model for Vietnames and English text together?
I repair symbol character to 
_characters_   =    'AĂÂBCDĐEÊGHIKLMNOÔPQRSTUƯVXYFJWZ' \
                    'ÁẮẤÉẾÍÓỐỚÚỨÝ' \
                    'ÀẰẤÈỀÌÒỒỜÙỪỲ' \
                    'ẢẲẨẺỂỈỎỔỞỦỬỶ' \
                    'ÃẴẪẼỄĨÕỖỠŨỮỸ' \
                    'ẠẶẬẸỆỊỌỘỢỤỰỴ' \
                    'aăâbcdđeêghiklmnoôpqrstuưvxyfjwz' \
                    'áắấéếíóốớúứý' \
                    'àằấèềìòồờùừỳ' \
                    'ảẳẩẻểỉỏổởủửỷ' \
                    'ãẵẫẽễĩõỗỡũữỹ' \
                    'ạặậẹệịọộợụựỵ' \
                    '!\'(),-.:;? '
I hope that it is useful for multi language

**Comment by keithito** [comment]
Yes, I think that should work. I don't know much about Vietnamese, but as I understand it, some letters are the same as in the English alphabet, but may be pronounced differently. The model should be able to learn to figure this out based on context, it would be interesting to see how it works out.

**Comment by toannhu** [comment]
@tuong-olli Can you send some results of training Vietnamese with Tacotron? Recently I've got some problem with training Vietnamese.

**Comment by tuong-olli** [comment]
@toannhu I hope to help you something.

[eval.zip](https://github.com/keithito/tacotron/files/1520543/eval.zip)

**Comment by toannhu** [comment]
@tuong-olli Tks. The sound is really good. What is the corpus you are using? Can you share it to me?
